src/test/java/com/taf/tests/BaseTest.java

```diff
@@ -0,0 +1,29 @@
+package com.taf.tests;
+
+import com.codeborne.selenide.Configuration;
+import com.codeborne.selenide.Selenide;
+import com.taf.core.ConfigManager;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+
+import static com.codeborne.selenide.Selenide.open;
+
+class BaseTest {
+
+    @BeforeAll
+    public static void setUp() {
+        Configuration.browser = "chrome";
```

Comment:
Browser type also better pass as a parameter

Reply:
Thank you for the advice, done.